hack/kind-up.sh

```diff
@@ -39,7 +39,11 @@ parse_flags() {
 }
 
 setup_loopback_device() {
-  if ! command -v ip &> /dev/null; then
+  if ! command -v ip &>/dev/null; then
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+      echo "'ip' command not found. Please install 'ip' command using 'brew install iproute2mac'" 1>&2
```

Comment:
Nit: Instead of printing the command, can we just link this doc:
https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md#installing-iproute2